webapp/src/Controller/Jury/ContestController.php

```diff
@@ -66,7 +66,7 @@ public function indexAction(Request $request): Response
     {
         $em = $this->em;
 
-        if ($doNow = (array)$request->request->get('donow')) {
+        if ($doNow = $request->request->all('donow')) {
```

Comment:
Just to be sure, the naming of the function does not reflect what actually happens? So we still only get 1 parameter?

Reply:
It gets *all* donow array values